resources/views/admin/categories.blade.php

```diff
@@ -0,0 +1,21 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-md-6">
+                <form>
+                    <div class="form-group">
+                      <label for="title">Title</label>
+                      <input type="text" class="form-control" id="title" name="title">
+                    </div>
+                    <div class="form-group">
+                        <label for="description">Description</label>
+                        <textarea name="desctiption" id="description"rows="3" style="min-width:100%;"></textarea>
```

Comment:
Please fix the spelling here, it might cause you errors later when you make the function.
```suggestion
                        <textarea name="description" id="description"rows="3" style="min-width:100%;"></textarea>
```